source/_integrations/fastdotcom.markdown

```diff
@@ -25,62 +26,12 @@ If you want to measure bandwidth metrics other then download such as ping and up
 
 Enabling this integration will automatically create the Fast.com Sensor.
 
-By default, a speed test will be run every hour. The user can change the update frequency in the configuration by defining the `scan_interval` for a speed test to run.
+By default, a speed test will be run every hour. The user can manually run a speed test via the `homeassistant.update_entity` service.
```

Comment:
I don't think that's the case as it's not using the `DataUpdateCoordinator`. I think they need to use the manual `update` service which is created by the integration

Reply:
It uses the `async_track_time_interval(hass, data.update, timedelta(hours=DEFAULT_INTERVAL))` to retry every hour.